signintokens/client.go

```diff
@@ -0,0 +1,51 @@
+// Package session provides the Sign In Token API.
+package signintokens
```

Comment:
:wrench: Let's change the package name (and the folder) to `signintoken`. Most of the package names in the library use singular form for the resources.